workers/unity/Assets/Playground/Scripts/Worker/IMobileConnectionController.cs

```diff
@@ -0,0 +1,11 @@
+namespace Playground
+{
+    public interface IMobileConnectionController
```

Comment:
Can this not be rolled into the mobile connector?

Reply:
the main issue here is that this needs to be used by both worker connectors. However, as it is playground specific we don't to move it into the `MobileWorkerConnector`.

We have the following options:
- expose this interface
- move it into the specific types and do a type check in the `ScreenController` to identify the exact type
- have a screen controller for each platform which contain the exact some logic, but would get rid of the interface and the type checks

We decided to go with the interface for this one as it made sense for this specific case

Reply:
Would it help to make the name highlight it's only for the playground? May be overkill though

Reply:
`IMobileConnectionControllerPlaygroundOnly` 😂

Reply:
💯